sphinx/writers/latex.py

```diff
@@ -355,7 +355,10 @@ def __init__(self, document: nodes.document, builder: LaTeXBuilder,
                 sphinxpkgoptions.append('nonumfigreset')
 
         if self.config.numfig and self.config.math_numfig:
-            sphinxpkgoptions.append('mathnumfig')
+            sphinxpkgoptions.extend([
+                'mathnumfig',
+                'mathnumsep=%s' % self.config.math_numsep,
```

Comment:
Hopefully the value will never need LaTeX escaping...

The `self.config.math_numsep` defaults to `.`? (I need to check) If user has not used it the injection of the config could be dropped for LaTeX, because the `\DeclareStringOption` has declared `.` as fall-back.

Reply:
Yes, `self.config.math_numsep` defaults to `.`.